src/windows/win-crowdstrike-fix-bootloop-v2.ps1

```diff
@@ -75,15 +75,48 @@ function LoadUnloadRegistryHives
                     $regFile = $regKeyToFile[$regKey]
                     if (Test-Path -Path $regFile) {
                         Log-Info "Found registry config file at $regFile."
-                        $configExist = $true
+                        $configExistInDrive = $true
 
                         Log-Info "Loading registry hive $regKey from $regFile..."
+                        $succeeded = $false
                         $result = reg load $regKey $regFile 2>&1
                         if ($LASTEXITCODE -ne 0) {
                             Log-Error "Load registry hive $regKey from $regFile failed with exit code $LASTEXITCODE. Error: $result"
-                        } else {
+                            if ($result -Match "corrupt") {
```

Comment:
Do we have confirmation to run this only when reg load fails with "corrupt" message?

Reply:
The #5 below from Chris indicates this is the only scenario.
![image](https://github.com/user-attachments/assets/38097a36-8ab6-4774-a496-cefadfafdd09)


I have queried all the error log of the script execution of the last few days' data. This is the only error message I saw so far.